test/test_game.py

```diff
@@ -335,7 +335,7 @@ def test_multiple_enemies_killing():
     ########
     """
     # dummy bots
-    stopping = lambda bot, s: (bot.position, s)
+    stopping = lambda bot, s: bot.position
```

Comment:
The stopping players ‘worked’ because they were only used as dummies to get through `setup_game`. The game was then simulated with `apply_move` without ever calling the function. I suppose we could have used a more obvious dummy for this.

Reply:
No, no, `apply_move` was called in some tests and the check `if bot_position not in legal_moves` was failing, because `(position, {})` could not be found in the list of tuples `legal_positions`. This was filing an error for the team. The simulated games only had one round, though, so the error never became fatal and all other checks in the tests were still correct.
they started failing for me when I transformed the check `if bot_position not in legal_moves` into a fatal thing. Then the tests couldn't proceed and would fail early.

Reply:
Yes, in some tests it failed. But in most others it didn’t. Doesn’t really matter but we can refactor the tests anyway at some point.